samples/bluetooth/hci_lpuart/README.rst

```diff
@@ -42,6 +42,12 @@ Testing
 
 The methodology to use to test this sample depends on the host application.
 
+Tips
+****
+
+This sample is also supported on the Thingy:91. However, it must be programmed using a debugger and a 10-pin SWD cable.
+Firmware updates over serial using MCUBoot is not supported for either of the MCUs on the Thingy:91 in this configuration.
+
```

Comment:
I think we can make this as part of the overview section above in both the samples.
```suggestion
This sample is also supported on the Thingy:91.
However, it must be programmed using a debugger and a 10-pin SWD cable.
Firmware updates over serial using MCUboot is not supported for either of the MCUs on the Thingy:91 in this configuration.

```